python/python/lance/torch/data.py

```diff
@@ -42,9 +45,10 @@ def _to_tensor(
         if pa.types.is_uint64(arr.type) and uint64_as_int64:
             arr = arr.cast(pa.int64())
 
-        if pa.types.is_fixed_size_list(arr.type) and pa.types.is_floating(
-            arr.type.value_type
-        ):
+        if (
+            pa.types.is_fixed_size_list(arr.type)
```

Comment:
We should just bump arrow and release lance 0.9 :)

Reply:
This set of cosine change is breaking change anyway, as the previous built cosine index can not be used anymore.

Reply:
Ok sounds good